### PR TITLE
[AIRFLOW-1247] Fix ignore all dependencies argument ignored

### PR DESCRIPTION
**Pull request by ghost**
Cause: Comma missing so the store_true argument goes to help string.
Fix: Add missing comma

Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [ ] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-XXX

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by aoen** [comment]
LGTM, thanks!

**Comment by bolkedebruin** [comment]
Can you please add tests and update the commit message?

**Comment by bolkedebruin** [comment]
@mremes please don't forget the tests: it is the reason why it was not caught in the first place.

**Comment by ghost** [comment]
@bolkedebruin added tests for upstream and depends_on_past dependencies not met. Run the scenario locally as I don't have the dev environment setup for running the tests yet... Will probably need to as there's more to contribute

**Comment by bolkedebruin** [comment]
can you make the tests pass?

**Comment by ghost** [comment]
@bolkedebruin It passes all well

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -1315,7 +1315,7 @@ class CLIFactory(object):
         'ignore_all_dependencies': Arg(
             ("-A", "--ignore_all_dependencies"),
             "Ignores all non-critical dependencies, including ignore_ti_state and "
-            "ignore_task_deps"
+            "ignore_task_deps",
```
Comment (saguziel) [inline airflow/bin/cli.py:1318]: better to change this to action="store_true", then this issue would have been avoided. Bonus points for fixing the othoer cases.

Comment (ghost) [reply airflow/bin/cli.py:1318]: @saguziel probably a refactoring PR would be the right place to do that.

**Comment by aoen** [comment]
@mremes  Ready to LGTM/merge when tests are passing: https://travis-ci.org/apache/incubator-airflow/builds/237275876?utm_source=github_status&utm_medium=notification

**Comment by ashb** [comment]
@mremes One of the causes of the test failure is a missing license in `dags/test_cli_triggered_dags.py` -- could you add it please?

**Comment by criccomini** [comment]
Checking on status. If you want this in 1.9.0, it needs to get merged shortly.

**Comment by mremes-ng** [comment]
@criccomini done, fixed tests by adding the licence to the head of the test file.

The code change is the same as in the https://github.com/apache/incubator-airflow/pull/2441, except this adds the tests for the functionality.

**Comment by ghost** [comment]
@criccomini done, fixed tests by adding the license to the head of the test file.

The code change is the same as in the #2441, except this adds the tests for the functionality.

**Comment by criccomini** [comment]
Merging!

**Pull request closed without merging**